polaris-react/src/components/DropZone/DropZone.scss

```diff
@@ -68,13 +75,23 @@ $dropzone-stacking-order: (
   }
 
   &:not(.isDisabled) {
+    #{$se23} &::after {
```

Comment:
This is to ensure we override the `border-color: var(--p-color-border-strong);` declaration on line 73. 
If I'm honest, not entirely sure why this is necessary, since line 104 and the subsequent se23 override on line 108 seems to catch all the cases, but adding it in here just in case there's an edge case that's not obvious.